deluge/plugins/Stats/deluge/plugins/stats/gtkui.py

```diff
@@ -66,22 +72,22 @@ def int_str(number):
 
 
 def gtk_to_graph_color(color):
-    """Turns a gtk.gdk.Color into a tuple with range 0-1 as used by the graph"""
-    max_val = float(65535)
-    gtk_color = gtk.gdk.Color(color)
-    red = gtk_color.red / max_val
-    green = gtk_color.green / max_val
-    blue = gtk_color.blue / max_val
+    """Turns a Gdk.Color into a tuple with range 0-1 as used by the graph"""
```

Comment:
Would [gdk_rgba_to_string](https://developer.gnome.org/gdk3/stable/gdk3-RGBA-Colors.html#gdk-rgba-to-string) be a better replacement here?